JPS_Ontology/ontology/ontodevice/OntoDevice.owl

```diff
@@ -1106,7 +1094,7 @@
     <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontodevice/ActivityType">
         <rdfs:comment>Represents the type of activity being performed (e.g.</rdfs:comment>
         <rdfs:isDefinedBy rdf:resource="walking"/>
-        <rdfs:label> biking) defined by Google&apos;s activity recognition API.</rdfs:label>
+        <rdfs:label> biking).</rdfs:label>
```

Comment:
There is something wrong with this section here (the lines for rdfs:isDefinedBy and rdfs:label). After addressing the other comments, try to regenerate this owl file and check whether this section is still the same.

Reply:
Probably it's caused by the description of ActivityType: 
Represents the type of activity being performed (e.g., walking, biking).